### DISCUSSION TITLE
The loss doesnot drop beyond e-3 in case of hPINN with only BCs in loss Function

### DISCUSSION THREAD

**Discussion by ppandit95** [lululxvi/deepxde#972] [Q&A] [answered]
Dear LuLu,

I am trying to solve inverse problem to train the interdiffusion coefficients by considering Fick's First Law as PDE in hPINN setting but I have functions defined for boundary conditions(not exactly at the boundary but a function which approximates the observed data spread throughout the 1D domain) and as a first step I am trying to use operatorBC in loss function along with some other constraints on interdiffusion coefficients  and setting weights for all other values to 0 while keeping only boundary conditions defined through operatorBC but I am able to reduce my MAE loss function just up to the order of e-3 which although approximates the output of my neural network but can this be further improved.Any suggestions in this regard will prove beneficial

Many Thanks
Pushkar

**Comment by lululxvi** [comment, marked as answer]
For inverse problems, you don't need hPINN. hPINN is for inverse design.

**Reply by ppandit95** [reply]
Thanks for your comments on and as you said I did manage to solve inverse problem without hPINN.....
